recipes/opentelemetry-cpp/all/conanfile.py

```diff
@@ -61,6 +62,7 @@ class OpenTelemetryCppConan(ConanFile):
         "with_logs_preview": False,
         "with_async_export_preview": False,
         "with_metrics_exemplar_preview": False,
+        "with_examples": True,
```

Comment:
hi @jiaoew1991  - thanks for opening this PR. 

We typically exclude building examples, most recipes where the underlying build scripts build examples by default would have them explicitly disabled.  The rationale is that they are _not_ part of the public API of the library and thus not necessary for consumers, and the Conan test package serves as a minimal example to cover this - Is it not the case with `opentelemetry-cpp`?

Trying to understand the use

Reply:
I want to disable the examples too, but in the original recipe the example was not disabled and the cmake of  `opentelemetry-cpp` build the examples by default. I do not want to break the compatibility so keep building the example 😅